ggml.c

```diff
@@ -18854,7 +19467,6 @@ static enum ggml_opt_result linesearch_backtracking(
                     // strong Wolfe condition (GGML_LINESEARCH_BACKTRACKING_STRONG_WOLFE)
                     return count;
                 }
-                return count;
```

Comment:
Curious if fixing this bug has any positive effect on the poor L-BFGS results that you reported earlier